docs/topics/Gateway.md

```diff
@@ -1111,6 +1116,15 @@ Active sessions are indicated with an "online", "idle", or "dnd" string per plat
 | SYNC         | 1 << 4 |
 | PLAY         | 1 << 5 |
 
+###### Activity Buttons
+
+When received over the gateway, the `buttons` field is an array of strings, which are the button labels. Bots cannot access a user's activity button URLs. When sending, the `buttons` field must be an array of the below object:
+
+| Field | Type   | Description                             |
+|-------|--------|-----------------------------------------|
+| label | string | the text shown on the buttons           |
+| url   | string | the url opened when clicking the button |
```

Comment:
```suggestion
| Field | Type   | Description                                        |
|-------|--------|----------------------------------------------------|
| label | string | the text shown on the button                       |
| url   | string | 1-32 character url opened when clicking the button |
```
`url`s have a limit of 32 characters

Reply:
they are 512, but the client validation was set to 32. it will be fixed in next deploy (January)

Reply:
ah, sounds nice